src/dialogs/more-info-dialog.html

```diff
@@ -28,6 +28,25 @@
         font-size: 14px;
       }
 
+      @@media all and (min-width: 451px){
+        paper-dialog[data-domain=alarm_control_panel],
```

Comment:
Instead of trying to aim at all domains, do it the other way around:

```css
paper-dialog {
  width: 450px;
}

paper-dialog[data-domain=camera] {
  min-width: 450px; /* or whatever seems reasonable */
}
```